apps/api/src/modules/resume/resume.service.ts

```diff
@@ -9,9 +13,27 @@ export class ResumeService {
 
   async createResume(postResumeDto: PostResumeDto, user: JwtDto) {
     const actor = await this.repository.getActorFromUser(user)
-    if (!actor) {
-      throw new NotFoundException('Actor not found')
-    }
     return this.repository.createResume(postResumeDto, actor, user.userId)
   }
+
+  async getResumeById(resumeId: number, user: JwtDto) {
+    const resume = await this.repository.getResumeById(resumeId)
+    if (!resume) {
+      throw new NotFoundException('Resume not found')
+    }
+    if (user.type === UserType.ACTOR) {
+      const actor = await this.repository.getActorFromUser(user)
+      if (resume.actorId !== actor.actorId) {
```

Comment:
```suggestion
      if (resume.actorId !== user.userId) {
```
You don't need to use actor here